exporter/datadogexporter/internal/model/internal/utils/tags_test.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package utils // import "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/datadogexporter/internal/model/internal/utils"
```

Comment:
What happened to this one?

Reply:
I think what happened is:
1. this file does not have the import path on the datadog-agent codebase and 
2. `goporto` doesn't add the import path to test files

so it was removed. Test files on this repository generally don't have the import path, so although it was not intentional, I think it's fine to leave it like this (unless you think otherwise)

Reply:
Test files don't need the comment.